### PR TITLE
Initial setup work

### PR DESCRIPTION
**Pull request by robinmitra**

### CODE REVIEW

File: upload.py [hunk 1]
```diff
@@ -1,18 +1,54 @@
-import os
```
Comment (donnabelsey) [inline upload.py]: The changes in upload.py don't seem to be related to setup; were these changes committed by mistake, or do they belong in a separate commit?

Comment (robinmitra) [reply upload.py]: Yes they were included in the commit by mistake.

File: Makefile [hunk 2]
```diff
@@ -49,17 +50,3 @@ test-flake8: virtualenv
 .PHONY: test-unit
 test-unit: virtualenv
 	${VIRTUALENV_ROOT}/bin/py.test ${PYTEST_ARGS}
-
-# Build and tag Docker image.
-.PHONY: docker-build
```
Comment (galund) [inline Makefile]: Let's not lose this tho eh :-)

Comment (robinmitra) [reply Makefile]: The reason for removing these was because they were incorrectly referring to digital marketplace's Docker Hub account, and we haven't really thought about how we're going to be using Docker for this project anyway. We can always bring back Docker specific commands later once we're at that stage.

File: upload.py (hunk 1 above)
Comment (robinmitra) [reply upload.py]: As discussed over chat, changes to this file have been moved to a separate commit, but within the same branch on this occasion.

File: Makefile (hunk 2 above)
Comment (galund) [reply Makefile]: Ah right I assuemed they were something you worked on the other day. All good anyway, def not for this PR.

Comment (donnabelsey) [reply Makefile]: We can fish it out of the git history if/when we need it later, it's fine
